qrisk/stats.py

```diff
@@ -166,67 +147,81 @@ def max_drawdown(returns):
 
     df_cum_rets = cum_returns(returns, starting_value=100)
     cum_max_return = df_cum_rets.cummax()
+    drawdown = df_cum_rets.sub(cum_max_return).div(cum_max_return)
 
-    return df_cum_rets.sub(cum_max_return).div(cum_max_return).min()
+    return drawdown.min()
 
 
-def annual_return(returns, period=DAILY):
-    """Determines the annual returns of a strategy.
+def annual_return(returns, period=DAILY, annualization=ANNUALIZATION_FACTORS):
+    """Determines the mean annual growth rate of returns.
 
     Parameters
     ----------
     returns : pd.Series
         Periodic returns of the strategy, noncumulative.
-         - See full explanation in tears.create_full_tear_sheet.
+        - See full explanation in cum_returns.
     period : str, optional
         - defines the periodicity of the 'returns' data for purposes of
         annualizing. Can be 'monthly', 'weekly', or 'daily'
         - defaults to 'daily'.
+    annualization : dict, optional
```

Comment:
The api here seems unintuitive. to pass a custom period, I would need to pass both the key and the dict so that the function call getitem for me. How do you feel about just passing the numeric value?